src/mappings/utils.ts

```diff
@@ -122,3 +129,28 @@ export async function trackUnprocessed(error: Error, primitives: Primitives): Pr
     logger.error(`[trackUnprocessable] (ERROR | stack): ${error.stack}`);
   }
 }
+
+export function getBalanceId(address: string, denom: string): string {
```

Comment:
Would you mind adding a comment like you have for `getEventId()`?

Reply:
Done!